### PR TITLE
Kubernetes feature tracking issue template

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

**Comment by pwittrock** [comment]
cc @bgrant0607  @quinton-hoole

**Comment by erictune** [comment]
LGTM.  We can iterate on it.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Large features typically go through three stages: [Alpha, Beta, and Stable](https://github.com/kubernetes/kubernetes/blob/master/docs/api.md#api-versioning)
+Each stage requires various approvals from various teams.
+
+You can use the checkboxes below to
+track your progress.  You can delete the text above once you have read it.  Please keep the checklist.
+
+- [ ] Before Alpha
+    - [ ] Design Approval
+      - [ ] < 1 page PR description of the design. [advice for design description]  **PR-NUMBER**
```
Comment (timstclair) [inline ISSUE_TEMPLATE.md]: FYI, you're missing a link here.

**Comment by harshanarayana** [comment]
Hey there  -- 1.19 Enhancements shadow here. I wanted to check in and see if you think this Enhancement will be graduating in 1.19?

In order to have this part of the release:
1. The KEP PR must be merged in an `implementable` state
2. The KEP must have test plans
3. The KEP must have graduation criteria.

The current release schedule is:
- Monday, April 13: Week 1 - Release cycle begins
- Tuesday, May 19: Week 6 - Enhancements Freeze
- Thursday, June 25: Week 11 - Code Freeze
- Thursday, July 9: Week 14 - Docs must be completed and reviewed
- Tuesday, August 4: Week 17 - Kubernetes v1.19.0 released


If you do, I'll add it to the 1.19 tracking sheet (http://bit.ly/k8s-1-19-enhancements). Once coding begins please list all relevant k/k PRs in this issue so they can be tracked properly. 👍

Thanks!

**Comment by harshanarayana** [comment]
Hey there  -- 1.19 Enhancements shadow here. I wanted to check in and see if you think this Enhancement will be graduating in 1.19?

In order to have this part of the release:
1. The KEP PR must be merged in an `implementable` state
2. The KEP must have test plans
3. The KEP must have graduation criteria.

The current release schedule is:
- Monday, April 13: Week 1 - Release cycle begins
- Tuesday, May 19: Week 6 - Enhancements Freeze
- Thursday, June 25: Week 11 - Code Freeze
- Thursday, July 9: Week 14 - Docs must be completed and reviewed
- Tuesday, August 4: Week 17 - Kubernetes v1.19.0 released


If you do, I'll add it to the 1.19 tracking sheet (http://bit.ly/k8s-1-19-enhancements). Once coding begins please list all relevant k/k PRs in this issue so they can be tracked properly. 👍

Thanks!

**Pull request merged**